src/applications/mhv-landing-page/containers/App.jsx

```diff
@@ -35,6 +35,7 @@ const App = () => {
   const unreadMessageAriaLabel = resolveUnreadMessageAriaLabel(
     unreadMessageCount,
   );
+  const mhvAccount = useSelector(state => state.mhvAccount);
```

Comment:
I'd like to recommend creating a selector that returns `state?.mhvAccount?.termsAndConditionsAccepted || false`, placing it in `src/applications/mhv-landing-page/selectors/` and writing a unit test for it.

Reply:
thank you, will do